imgui-sys/src/lib.rs

```diff
@@ -785,7 +785,7 @@ extern "C" {
     pub fn igRadioButton(label: *const c_char, v: *mut c_int,
                          v_button: c_int) -> bool;
     pub fn igCombo(label: *const c_char, current_item: *mut c_int,
-                   items: *mut *const c_char, items_count: c_int, height_in_items: c_int) -> bool;
+                   items: *const *const c_char, items_count: c_int, height_in_items: c_int) -> bool;
```

Comment:
I'm a bit pedantic here, but this change is not correct.
A `const char** items` has only one level of const. It is true that ImGui will not modify items so a `*mut` pointer is unnecessary, but I'd prefer wrapping stuff exactly as-is to match the level of const correctness in the original library